WordPress/Classes/Services/BlogService.m

```diff
@@ -659,8 +659,13 @@ - (OptionsHandler)optionsHandlerWithBlogObjectID:(NSManagedObjectID *)blogObject
                 return;
             }
             blog.options = [NSDictionary dictionaryWithDictionary:options];
-            blog.siteVisibility = (SiteVisibility)([[blog getOptionValue:@"blog_public"] integerValue]);
-            blog.isAdmin = YES;
+			blog.siteVisibility = (SiteVisibility)([[blog getOptionValue:@"blog_public"] integerValue]);
```

Comment:
Annoying comment: can we please please normalize left spacing on this one?

(Sorry!!)